src/views/layout/PageLayout/PageLayoutTs.ts

```diff
@@ -275,8 +275,6 @@ export class PageLayoutTs extends Vue {
     }
 
     public reconnect() {
-        if (this.$route.fullPath !== '/home') {
-            this.$store.dispatch('network/CONNECT', { waitBetweenTrials: true });
-        }
+        this.$store.dispatch('network/CONNECT', { waitBetweenTrials: true });
```

Comment:
I would suggest adding if condition in case the current path is`/login` so the wallet wouldn't try to connect to any of the nodes while users are not logged in yet.

Reply:
I checked the `reconnect` method only able triggers when `current_node_is_not_available` alert appears.

Does the login page will have this alert as well?

Reply:
nope, not for the alert but we got a watcher in that class that could retry to re-connect the wallet even if users are not logged in.